sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/TimestampFormatter.scala

```diff
@@ -121,6 +121,7 @@ class FractionTimestampFormatter(zoneId: ZoneId)
     TimestampFormatter.defaultPattern,
     zoneId,
     TimestampFormatter.defaultLocale,
+    LegacyDateFormats.FAST_DATE_FORMAT,
```

Comment:
is it the same with Spark 2.4?

Reply:
Spark 2.4 uses SimpleDateFormat but output of FastDateFormat and SimpleDateFormat for the concrete pattern `yyyy-MM-dd HH:mm:ss` is the same. At least, all our tests show no difference.